openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -4726,6 +4726,87 @@ OS:ScheduleTypeLimits,
        \key ControlMode
        \key LinearPowerDensity
 
+OS:External:File,
+       \min-fields 3
+       \memo An External:File points to a text computer file for use in Schedule:File objects.
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Name
+       \required-field
+       \type alpha
+       \reference ExternalFileNames
+  A3,  \field File Name
+       \required-field
+       \retaincase
+  A4;  \field Column Separator
+       \type choice
+       \key Comma
+       \key Tab
+       \key Space
+       \key Semicolon
+       \default Comma
+
+OS:Schedule:File,
+       \min-fields 6
+       \memo A Schedule:File points to a text computer file that has 8760-8784 hours of data.
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Name
+       \required-field
+       \type alpha
+       \reference ScheduleNames
+  A3,  \field Schedule Type Limits Name
+       \type object-list
+       \object-list ScheduleTypeLimitsNames
+  A4,  \field File Name
+       \required-field
+       \object-list ExternalFileNames
+  N1,  \field Column Number
+       \required-field
+       \type integer
+       \minimum 1
+  N2,  \field Rows to Skip at Top
+       \required-field
+       \type integer
+       \minimum 0
+  N3,  \field Number of Hours of Data
+       \note 8760 hours does not account for leap years, 8784 does.
+       \note should be either 8760 or 8784
+       \default 8760
+       \minimum 8760
+       \maximum 8784
+  A5,  \field Column Separator
+       \type choice
+       \key Comma
+       \key Tab
+       \key Fixed
+       \key Semicolon
+       \default Comma
+  A6,  \field Interpolate to Timestep
+       \note when the interval does not match the user specified timestep a "Yes" choice will average between the intervals request (to
+       \note timestep resolution.  a "No" choice will use the interval value at the simulation timestep without regard to if it matches
+       \note the boundary or not.
+       \type choice
+       \key Yes
+       \key No
+       \default No
+  N4;  \field Minutes per Item
```

Comment:
Based on \min-fields 6 this field should have a default value, is the default 60?